### ISSUE TITLE
Cannot build a docker image of oauth2-provider

### ISSUE THREAD

**Issue by mindaugasrukas** [zalando-incubator/mocks#34]
```
Step 8 : COPY /scm-source.json /
lstat scm-source.json: no such file or directory
```

.gitignore contains:
https://github.com/zalando-incubator/mocks/blob/master/.gitignore#L4

**Comment by hjacobs** [comment]
Just run `scm-source` (`pip3 install scm-source`) in the current dir. See http://docs.stups.io/en/latest/user-guide/application-development.html#scm-source-json

Needs to be added to README.

**Comment by jans510** [comment]
We'll add it to the README, thanks for pointing it out.

**Comment by jans510** [comment]
Fixed with PR #35
